internal/attacktechniques/aws/exfiltration/ec2-share-ebs-snapshot/main.tf

```diff
@@ -22,9 +22,16 @@ data "aws_availability_zones" "available" {
   state = "available"
 }
 
+resource "aws_kms_key" "cmk" {
+  description = "Stratus Red Team CMK for EBS encryption"
+  deletion_window_in_days = 7
```

Comment:
Although I don't have a better solution in mind, I'm slightly concerned that the key remains in the account for several days, potentially incurring costs and confusion

Reply:
I agree it's less than ideal and a bit messy leaving keys around. The more elegant solution would be to allow conditional arguments or global configuration to provide a single CMK for all operations only if needed

KMS keys cost $1/month but don't incur any cost while pending deletion:
> There is no charge for customer managed KMS keys that are scheduled for deletion. If you cancel the deletion during the waiting period, the customer managed KMS key will incur charges as though it was never scheduled for deletion.